src/app/admin/volunteers/volunteers-details/volunteers-details.component.ts

```diff
@@ -21,12 +29,27 @@ export class VolunteersDetailsComponent implements OnInit, OnDestroy {
     private route: ActivatedRoute,
     private volunteerFacade: VolunteersFacade,
     private translateService: TranslateService,
-  ) {}
+  ) {
+    this.route.paramMap
+      .pipe(
+        map((params) => params.get('id') as string),
+        tap((id) => (this.recordId = id)),
```

Comment:
redundant line, is assigned in `subscribe`
Maybe can be replaced `tap` into `filter` to filter when id is not defined in any case, and bellow in `subscribe` `if` wrapper can be removed